catroid/src/main/java/org/catrobat/catroid/content/actions/RepeatAction.java

```diff
@@ -67,7 +67,7 @@ public boolean delegate(float delta) {
 		if (executedCount >= repeatCountValue && !isForeverRepeat) {
 			return true;
 		}
-		if (action.act(delta) && currentTime >= LOOP_DELAY) {
+		if (action != null && action.act(delta) && currentTime >= LOOP_DELAY) {
```

Comment:
> do you knw why this can be null?

e.g. [StopOtherScriptsAction](https://github.com/Catrobat/Catroid/blob/2b637e7fefb7c91614ad79c3fb504e91a101b8e0/catroid/src/main/java/org/catrobat/catroid/content/actions/StopOtherScriptsAction.java#L47) calls reset which sets the action to null [here (libgdx.DelegateAction)](https://github.com/libgdx/libgdx/blob/b0163a4eaf95aca6cb04d300c82bd3d13f142dee/gdx/src/com/badlogic/gdx/scenes/scene2d/actions/DelegateAction.java#L55).

When and where is the action set again? Or is the whole RepeatAction recreated?